CHANGES.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 1.8.11 (core) / 0.24.11 (libraries)
+
+### New
+
+- AutomationCondition.eager() will now only launch runs for missing partitions which become missing _after_ the condition has been added to the asset. This avoids situations in which the eager policy kicks off a large amount of work when added to an asset with many missing historical static/dynamic partitions.
+- Add AutomationCondition.asset_matches() capability.
+- Added the ability to experimentally specify multiple kinds for an asset with the `kinds` parameter.
+- [dagster-github] Added create_pull_request method on GithubClient that enables creating a pull request.
+- [dagster-github] Added `create_ref` method on `GithubClient` that enables creating a new branch.
+- [dagster-embedded-elt] dlt assets now generate column metadata for child tables.
+- [dagster-embedded-elt] dlt assets can now fetch row count metadata with `dlt.run(...).fetch_row_count()` for both partitioned and non-partitioned assets. Thanks [@kristianandre](https://github.com/kristianandre)!
+- [dagster-airbyte] relation identifier metadata is now attached to Airbyte assets.
+- [dagster-embedded-elt] relation identifier metadata is now attached to sling assets.
+- [dagster-embedded-elt] relation identifier metadata is now attached to dlt assets.
+
+### Bugfixes
+
+- `PartitionedConfig` objects can now return a `RunConfig` without causing a crash.
+- Corrected the AssetIn.**new** typing for the dagster_type argument.
```

Comment:
I think `AssetIn.__new__` needs to be in backticks to avoid it being treated as bold / converted to stars